apps/cms/src/modules/cms/utils.ts

```diff
@@ -0,0 +1,93 @@
+import { SettingsValue } from "@saleor/app-sdk/settings-manager";
```

Comment:
We also have the `/lib/cms/utils.ts` file.

I will parrot @lkostrowski and say:
1. Please ensure they must live in a separate `utils` file. Once we extract them from their functionality, we lose the context of where and how they are used.
2. If so, please split the `utils` based on their purpose, and change the file names (e.g. `dates.ts`).
3. If you could take the other `utils` file into account, that would be great.

Reply:
There were many legacy utils here, I just removed most of them.

Reply:
As for now I left only this file, second utils I removed.